CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ Changes
 --------------
 * [UI/Developer]: Updated to lodash v4.17.11 to fix security issue. (19.01.1)
 * [Admin]: Added message to add `irida.db.profile` param for Tomcat in docs and upgrading guide.
+* [Admin]: Made analysis task pool size configurable with `irida.workflow.analysis.threads`. (19.01.2)
```

Comment:
Can you add something to the UPGRADING.md file as well?

> *something something* new config value *something* default value is 4 *something something* helps with lots of running jobs here's the docs.

Reply:
Done in ddcd8acf884453cc72dbeba9fc2e9557e4314855